### PR TITLE
Add sign in/log out link to nav

### PR DESCRIPTION
**Pull request by amitch23**
## Description

There is a login/logout link in nav bar - shows "Sign in" when user is not logged in, "Log out" and email address when user is signed in. 

## Reviewer Notes

Is the user data best stored in ducks under `shared/User/`, or does it make more sense for it to be stored somewhere like `shared/AppWrapper/`? 

## Verification Steps

* [x] All tests pass.
* [x] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/blob/master/docs/backend.md#logging)
* [x] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/blob/master/docs/backend.md#querying-the-database-safely)
 have been satisfied.
* [x] There are no [aXe](https://www.deque.com/products/aXe/) warnings for UI.
* [ ] This works in IE.
* [ ] Any new third party client side dependencies (e.g., login.gov, google analytics, CDN libraries, etc.) have been communicated to @willowbl00.

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/155131915) for this change

## Screenshots
screen shot 2018-03-30 at 4 59 18 pm
screen shot 2018-03-30 at 4 59 08 pm

### CODE REVIEW

File: src/scenes/Landing/index.jsx [hunk 1]
```diff
@@ -23,10 +22,6 @@ export class Landing extends Component {
     return (
       <div className="usa-grid">
         <h1>Welcome!</h1>
-        <div>
-          <LoginButton />
```
Comment (tinyels) [inline src/scenes/Landing/index.jsx]: I'd leave this here

File: src/shared/User/Email.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { bindActionCreators } from 'redux';
+import { loadUserAndToken } from 'shared/User/ducks';
+
+export class Email extends Component {
+  componentDidMount() {
+    this.props.loadUserAndToken();
+  }
+  render() {
+    return (
+      <span>{this.props.isLoggedIn && <span>{this.props.email}</span>}</span>
```
Comment (tinyels) [inline src/shared/User/Email.jsx]: When we get the user profile story, we should consider making this a link to user profile.

File: src/scenes/Landing/index.jsx (hunk 1 above)
Comment (amitch23) [reply src/scenes/Landing/index.jsx]: ok, leaving it in

File: src/shared/User/Email.jsx (hunk 2 above)
Comment (amitch23) [reply src/shared/User/Email.jsx]: sounds good

**Pull request merged**